parlai/crowdsourcing/tasks/turn_annotations_static/hydra_configs/conf/example.yaml

```diff
@@ -11,20 +11,14 @@ mephisto:
     extra_source_dir: ${task_dir}/webapp/src/static
     units_per_assignment: 5
     onboarding_qualification: turn_annotations_static
-    annotation_buckets: ${task_dir}/task_config/annotation_buckets.json
+    annotations_config_path: ${task_dir}/task_config/annotations_config.json
     response_field: False
+    task_description_file: ${task_dir}/task_config/task_description.html
```

Comment:
It'd be good to add in a sample `task_description.html` file as well here. Basically, the principle behind all tasks in `parlai/crowdsourcing/tasks/` is that they should be immediately runnable on the command-line as-is by just calling the `run.py` file without any other args, in order to let the user quickly understand what the task looks like and how it works. However, this requires all config files specified in the default YAML file to exist in the `task_config/` folder

Reply:
added!